draft-ietf-scitt-architecture.md

```diff
@@ -578,9 +578,9 @@ Multi-tenant support can be enabled through the use of identifiers in the `iss`
 Registration Policies refer to additional checks over and above the Mandatory Registration Checks that are performed before a Signed Statement is accepted to be registered to the Append-only Log.
 
 Transparency Services MUST maintain Registration Policies.
-Transparency Services MUST maintain a list of trust anchors (see definition of trust anchor in {{-Glossary}}) to verify Issuers of Signed Statements.
+Transparency Services MUST maintain a list of trust anchors (see definition of trust anchor in {{-Glossary}}) to verify Issuers of Signed Statements, either separately, or inside Registration Policies.
```

Comment:
```suggestion
Transparency Services MUST maintain a list of trust anchors (see definition of trust anchor in {{-Glossary}}) in order to check the signatures of Signed Statements, either separately, or inside Registration Policies.
```

Reply:
The purpose of this change is to clarify that maintaining a standalone list of keys is not necessary, because a policy can fulfill the same role, with more nuance.

Reply:
The purpose of my proposed change is to eliminate confusion regarding interpretation of what 'verify Issuers' might mean.